packages/subnav/partials/MenuItemsOverflow/style.module.css

```diff
@@ -16,7 +16,7 @@
   position: absolute;
   right: 0;
   top: 100%;
-  width: 256px;
+  min-width: 256px;
```

Comment:
💭 One potential challenge here might be on small viewports.

I think the `256px` value here was set as a strict thing rather than a `min` to try to more explicitly express that the dropdown can't get much wider than `256px` on small viewports without causing breakage:

<img width="454" alt="CleanShot 2022-08-22 at 17 19 55@2x" src="https://user-images.githubusercontent.com/4624598/186021298-ab24d025-cf42-4081-8e6f-bf9342972c50.png">

That being said, maybe this is too prescriptive, and I suppose if an element within the dropdown is wider than `256px`, we might end up seeing breakage either way... and using `min-width` does seem to offer flexibility to do wider things on larger viewports with media queries... and I trust there's some i8n-related rationale here too...  so I'm on board with this, just wanted to flag in case that concern is relevant! ☺️

Reply:
Gah, good catch. Just to confirm I understand though, setting the `min-width` to 256px means that the dropdown will grow in width depending on the content. The breakage you're referring occurs when the dropdown expands to a width (due to longer strings within it) that exceeds that of the viewport. Does that sound right?

Ex:
<img width="519" alt="Screen Shot 2022-08-22 at 5 40 21 PM" src="https://user-images.githubusercontent.com/39201593/186043556-da374e35-7506-427b-bad3-84753fc40871.png">

The context here is that I'm swapping out the custom nav used on the internationalized microsites with the `Subnav` component. Unfortunately, the text for the French CTA (see above) is pretty long so the CTA is overflowing out of the dropdown and off of the page. Given the issue you've flagged and the fact that the French text _seems_ like it should be shorter, I'll try to track down someone on the i18n side to see if it makes sense to come up with shorter copy.

In the meantime, I'll revert my commit and merge Alex's approved changes. Thanks for taking a look!

Reply:
Oooh yeah dang these are some interesting edge cases... for that button, I _think_ the ideal scenario would have the button text wrap? 🤔 That's a hot take though, I'm not sure!